test/run.py

```diff
@@ -0,0 +1,218 @@
+import time
+import subprocess
+import os
+
+import conu
+
+
```

Comment:
For start, I'd suggest having one global variable backend = conu.DockerBackend().